aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -544,6 +559,38 @@ module aptos_framework::stake {
         };
     }
 
+    public fun initialize_pending_transaction_fee(framework: &signer) {
+        system_addresses::assert_aptos_framework(framework);
+
+        if (!exists<PendingTransactionFee>(@aptos_framework)) {
+            move_to(framework, PendingTransactionFee {
+                pending_fee_by_validator: ordered_map::new(),
+            });
+        }
+    }
+
+    public(friend) fun record_fee(
+        vm: &signer,
+        fee_distribution_addresses: vector<address>,
+        fee_amounts_octa: vector<u64>,
```

Comment:
Where is `fee_amounts_octa` coming from? Is it calculated by each txn (concurrently) and then past into the epilogue txn? Or is this coming out of some execution summary kinda structure that already exists?